<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>982</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>982</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Index Order</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* --- MAIN WINDOW BACKGROUND --- */
QMainWindow {
    background-color: black;
}

/* --- TABS --- */
QTabWidget::pane {
    border: 1px solid #333; /* Subtle dark border around content */
    top: -1px; /* Merges border with tab bar */
	background-color: black;
}
QTabBar::tab {
    background: #0d0d0d;
    color: #666; /* Darker grey for inactive tabs */
    padding: 10px 20px;
    font-family: &quot;Footlight MT&quot;;
    font-size: 14pt;
    border: 1px solid #333;
    border-bottom: none;
    border-top-left-radius: 5px;
    border-top-right-radius: 5px;
    margin-right: 2px;
}
QTabBar::tab:selected {
    background: black;
    color: white;
    border-top: 2px solid white; /* White line on active tab */
    border-bottom: 1px solid black; /* Blends with background */
}
QTabBar::tab:hover {
    color: #bbb; /* Light grey on hover */
}

/* --- LABELS (Headings) --- */
QLabel {
    color: white;
    font-family: &quot;Footlight MT&quot;;
}

/* --- BUTTONS (Global Style) --- */
QPushButton {
    font: 300 18pt &quot;Footlight MT&quot;;
   	border: 2px solid rgb(31, 31, 31);   /* You need a border to see the curve clearly */
	border-radius: 15px;      /* Higher number = rounder corners */
	padding: 5px 10px;        /* Adds space inside so text doesn't hit the curve */
	color: white;
	background-color: rgb(13, 13, 13);
}
QPushButton:hover {
    background-color: white; /* Invert colors on hover */
    border: 2px solid rgb(62, 62, 62);
}
QPushButton:pressed {
    background-color: #ccc; /* Light grey when clicked */
    color: black;
}

/* --- INPUT FIELDS &amp; LISTS --- */
QLineEdit, QListWidget {
    background-color: rgb(13, 13, 13);
    color: white;
    border: 2px solid #555;
    border-radius: 10px;
    font-family: &quot;Footlight MT&quot;;
    font-size: 14pt;
    padding: 5px;
    selection-background-color: white;
    selection-color: black;
}
QLineEdit:focus, QListWidget:focus {
    border: 2px solid white; /* White Highlight when typing */
}

/* --- RADIO BUTTONS --- */
QGroupBox {
    color: white;
    font-family: &quot;Footlight MT&quot;;
    font-size: 12pt;
    border: 1px solid #333;
    border-radius: 5px;
    margin-top: 20px;
}
QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px;
    color: white; 
}
QRadioButton {
    color: white;
    font-size: 14pt;
    font-family: &quot;Footlight MT&quot;;
    spacing: 10px;
}
QRadioButton::indicator {
    width: 14px;
    height: 14px;
    border-radius: 8px;
    border: 2px solid #777;
    background: transparent;
}
QRadioButton::indicator:checked {
    background-color: white; /* White Fill */
    border: 2px solid white;
}
QRadioButton::indicator:hover {
    border: 2px solid #aaa;
}

/* --- SCROLL BAR (Optional: Makes it dark instead of standard Windows grey) --- */
QScrollBar:vertical {
    border: none;
    background: black;
    width: 10px;
    margin: 0px;
}
QScrollBar::handle:vertical {
    background: #444;
    min-height: 20px;
    border-radius: 5px;
}
QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    height: 0px;
}

QComboBox {
    background-color: rgb(13, 13, 13);
    color: white;
    border: 2px solid rgb(31, 31, 31);
    border-radius: 10px;
    padding: 5px 15px; /* Extra padding on right for arrow */
    font-family: &quot;Footlight MT&quot;;
    font-size: 14pt;
}

QComboBox::drop-down {
    border: none;
    background: transparent;
    width: 30px;
}

/* The arrow icon */
QComboBox::down-arrow {
    image: none;
    border-left: 5px solid transparent;
    border-right: 5px solid transparent;
    border-top: 7px solid white; /* Draws a white triangle */
    margin-right: 10px;
}

/* The popup list */
QComboBox QAbstractItemView {
    background-color: rgb(13, 13, 13);
    color: white;
     border: 2px solid rgb(31, 31, 31);
    selection-background-color: white;
    selection-color: black;
    outline: none;
}

/* --- NEW CARD --- */

.overlay-container {
  position: fixed;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  /* This creates a perfect center axis */
  display: flex;
  justify-content: center;
  align-items: center;
}

.new-order-card {
  position: fixed;
  top: 50%;
  left: 50%;
  /* This pulls the element back by half its own width/height to center it perfectly */
  transform: translate(-50%, -50%);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 300 12pt &quot;Footlight MT&quot;;
background-color: rgb(0, 0, 0);
</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Home</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>981</width>
          <height>151</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 300 32pt &quot;Footlight MT&quot;;</string>
        </property>
        <property name="text">
         <string>【 Index Order 】</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_generate">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>420</y>
          <width>311</width>
          <height>61</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 300 20pt &quot;Footlight MT&quot;;
background-color: rgb(13, 13, 13);
</string>
        </property>
        <property name="text">
         <string>New Order ///</string>
        </property>
       </widget>
       <widget class="QComboBox" name="combo_gen_mode">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>500</y>
          <width>291</width>
          <height>41</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Any</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Combat</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Narrative</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <property name="layoutDirection">
        <enum>Qt::LayoutDirection::LeftToRight</enum>
       </property>
       <attribute name="title">
        <string>Prescript Settings</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 300 32pt &quot;Footlight MT&quot;;</string>
          </property>
          <property name="text">
           <string>【 Prescripts 】</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LayoutDirection::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 0, 0);
border: none;</string>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>962</width>
             <height>394</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Policy::Maximum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>150</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QListWidget" name="list_orders">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
font: 300 14pt &quot;Footlight MT&quot;;
border-left: 2px dotted white;
padding-left: 10px;</string>
              </property>
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::SelectionMode::ExtendedSelection</enum>
              </property>
              <property name="isWrapping" stdset="0">
               <bool>false</bool>
              </property>
              <property name="resizeMode">
               <enum>QListView::ResizeMode::Adjust</enum>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Policy::Maximum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>150</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border: none;</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Policy::Maximum</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>150</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLineEdit" name="input_new_order">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 300 12pt &quot;Footlight MT&quot;;
border: 2px solid #555;   /* You need a border to see the curve clearly */
border-radius: 10px;      /* Higher number = rounder corners */
padding: 5px 10px;        /* Adds space inside so text doesn't hit the curve */
</string>
             </property>
             <property name="placeholderText">
              <string>Type your 'Prescript' here...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="combo_add_mode">
             <item>
              <property name="text">
               <string>Combat</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Narrative</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Policy::Maximum</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>150</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border: none;</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btn_delete">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>400</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 300 15pt &quot;Footlight MT&quot;;
border: 2px solid rgb(31, 31, 31);   /* You need a border to see the curve clearly */
border-radius: 15px;      /* Higher number = rounder corners */
padding: 5px 10px;        /* Adds space inside so text doesn't hit the curve */
color: white;
background-color: rgb(13, 13, 13);</string>
             </property>
             <property name="text">
              <string>Delete Selected Prescript</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btn_add">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>400</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 300 15pt &quot;Footlight MT&quot;;
border: 2px solid rgb(31, 31, 31);   /* You need a border to see the curve clearly */
border-radius: 15px;      /* Higher number = rounder corners */
padding: 5px 10px;        /* Adds space inside so text doesn't hit the curve */
color: white;
background-color: rgb(13, 13, 13);</string>
             </property>
             <property name="text">
              <string>Add a Prescript</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
